x-pack/plugins/file_data_visualizer/kibana.json

```diff
@@ -10,7 +10,8 @@
     "embeddable",
     "share",
     "discover",
-    "fileUpload"
+    "fileUpload",
+    "home"
```

Comment:
I think `home` should be an optional dependency here.

Reply:
Updated in https://github.com/elastic/kibana/pull/100863/commits/b532118bd2a9a35510b3a5ca1389498de89f1fdd